crypto-core/src/test/java/com/palantir/crypto/cipher/AbstractDecryptingInputStreamTest.java

```diff
@@ -57,35 +51,26 @@
 
     @BeforeClass
     public static void beforeClass() throws IOException {
-        fs.initialize(URI.create("file:///"), new Configuration());
-
         data = new byte[NUM_BYTES];
         random.nextBytes(data);
     }
 
     @Before
     public final void before() throws IOException {
-        // Only write data once. Cannot be in beforeClass since getSeekableCipher cannot be made static
-        if (!dataWritten) {
-            seekableCipher = getSeekableCipher();
-            path = new Path(tempFolder.newFile().getAbsolutePath());
-            FSDataOutputStream fos = fs.create(path);
-            OutputStream os = new FsCipherOutputStream(fos, seekableCipher);
-            os.write(data);
-            os.close();
-            dataWritten = true;
-        }
-
-        FSDataInputStream is = fs.open(path);
-        cis = new FsCipherInputStream(is, seekableCipher);
+        seekableCipher = getSeekableCipher();
```

Comment:
less than ideal to encrypt before every test, but getSeekableCipher isn't static